types/src/event.rs

```diff
@@ -64,6 +65,15 @@ impl From<&EventKey> for [u8; EventKey::LENGTH] {
     }
 }
 
+impl From<&EventKey> for AccountAddress {
+    fn from(e: &EventKey) -> Self {
+        let mut arr_bytes = [0u8; AccountAddress::LENGTH];
+        arr_bytes.copy_from_slice(&e.0[EventKey::LENGTH - AccountAddress::LENGTH..]);
+
+        AccountAddress::new(arr_bytes)
+    }
+}
+
```

Comment:
it's not really a conversion, without too much convenience gain by implementing `From`, I'd rather have a `EventKet::get_account_address()`

Reply:
done!